rust_src/src/xfaces.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{
+    lisp::defsubr,
+    remacs_sys::{clear_face_cache, windows_or_buffers_changed},
+};
+use remacs_macros::lisp_fn;
+use std::sync::atomic::{AtomicBool, Ordering};
+
+lazy_static! {
+    static ref FACE_CHANGE: AtomicBool = AtomicBool::new(false);
+}
+
+/// Clear face caches on all frames.
+/// Optional THOROUGHLY non-nil means try to free unused fonts, too.
+#[lisp_fn(name = "clear-face-cache")]
```

Comment:
There are two issues here. The first is why the compilation breaks. As written, this exports a function called `Frust_clear_face_cache`. To fix this `cname` also needs to be set. The other issue is `thoroughly` is meant to be optional. This means settings `min = "0"`.